src/component/flights/flights.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import './flights.css'
+import { Link } from 'react-router-dom';
+
+const style = {
+  marginLeft: "18px",
+  fontSize: "20px",
+  color: "white",
+  fontWeight: "bold"
+};
+const rightstyle = {
+  marginLeft: "18px",
+  fontSize: "20px",
+  color: "white",
+  textAlign: "right"
+};
+
+const button = {
+  backgroundColor: "#3C5E83",
+  border: "none",
+  color: "white",
+  padding: "6px 25px",
+  textAlign: "center",
+  display: "inline-block",
+  marginLeft: "390px",
+  borderRadius: "5px",
+  fontSize: "20px"
+};
+
+const header = {
+  textAlign: "center",
+  marginLeft: "18px",
+  fontSize: "40px",
+  color: "white"
+};
```

Comment:
This code should be moved to the existing flight.css file

Reply:
I will make all the style changes to css files and implement the right folders after the demo. Thanks for the suggestion